zetaclient/bitcoin_client.go

```diff
@@ -483,6 +486,10 @@ func FilterAndParseIncomingTx(txs []btcjson.TxRawResult, blockNumber uint64, tar
 						logger.Warn().Err(err).Msgf("error hex decoding memo")
 						continue
 					}
+					if bytes.Compare(memoBytes, []byte(DonationMessage)) == 0 {
```

Comment:
If memo contains "I am rich!" message then it's considered a donation transaction.